yaml/queueworker-dep.yml

```diff
@@ -21,7 +25,12 @@ spec:
           value: "30s"
         - name: faas_function_suffix
           value: ".openfaas-fn.svc.cluster.local."  # absolute DNS will suppress search domains
-        - name: basic_auth
-          value: "false"
         - name: secret_mount_path
-          value: "/etc/openfaas"
+          value: "/var/secrets"
+        - name: basic_auth
+          value: "{{ .Values.basic_auth }}"
```

Comment:
Some helm leftovers, replace with `true`

Reply:
I just pushed a fix for this. There must be something else stopping the alerts though.